projects/aca-shared/src/lib/components/document-base-page/document-base-page.spec.ts

```diff
@@ -24,18 +24,15 @@
 
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { PageComponent } from './document-base-page.component';
-import { AppState, ReloadDocumentListAction, SetSelectedNodesAction, ViewNodeAction } from '@alfresco/aca-shared/store';
+import { AppState, SetSelectedNodesAction, ViewNodeAction } from '@alfresco/aca-shared/store';
 import { AppExtensionService, LibTestingModule, discoveryApiServiceMockValue, DocumentBasePageServiceMock } from '@alfresco/aca-shared';
```

Comment:
Not changed in this PR but we should update the import statement. We are already in aca-shared

Reply:
it's the "@alfresco/aca-shared/store" library, another entrypoint

Reply:
I meant the import statement below it, at line num 28 :sweat_smile: 
```
import { AppExtensionService, LibTestingModule, discoveryApiServiceMockValue, DocumentBasePageServiceMock } from '@alfresco/aca-shared';
```